manual/manual/cmds/comp.etex

```diff
@@ -271,12 +271,12 @@ are two ways to fix this error:
 type (without type variables) to \var{name}. For instance, instead of
 writing
 \begin{verbatim}
-    let sort_int_list = Sort.list (<)
+    let sort_int_list = List.sort (<)
```

Comment:
This will give a type error. You should write: `List.sort Stdlib.compare` instead.

Same thing a few lines down.

Reply:
Fixed.